PrismSampleApp/PrismSampleApp/Views/LoginPage.xaml

```diff
@@ -44,8 +44,11 @@
                                 <CheckBox IsChecked="False" Color="HotPink" />
                                 <Label Text="Remember me" TextColor="DarkGray" FontSize="Small" VerticalTextAlignment="Center" HorizontalTextAlignment="Center" />
                                 <Label Text="Forgot Password" TextColor="HotPink" FontAttributes="Bold" HorizontalOptions="EndAndExpand" FontSize="Small" VerticalTextAlignment="Center" HorizontalTextAlignment="Center" />
+                                <Button Command="{Binding LoginCommand}" Text="SIGN IN" BackgroundColor="HotPink" TextColor="White" CornerRadius="30" />
```

Comment:
Add colors into app.xaml as a static resource and then use the same here, wherever you need to set colors